graylog2-server/src/main/java/org/graylog/events/processor/aggregation/AggregationEventProcessorConfig.java

```diff
@@ -194,37 +206,60 @@ public ValidationResult validate() {
         if (!series().isEmpty() && isConditionsEmpty()) {
             validationResult.addError(FIELD_CONDITIONS, "Aggregation with series must also contain conditions");
         }
+        return validationResult;
+    }
+
+    @Override
+    public ValidationResult validate(@Nullable EventProcessorConfig oldEventProcessorConfig,
+                                     EventDefinitionConfiguration eventDefinitionConfiguration) {
+        final ValidationResult validationResult = new ValidationResult();
+        AggregationEventProcessorConfig oldConfig = (AggregationEventProcessorConfig) oldEventProcessorConfig;
+
+        if (oldConfig == null) {
```

Comment:
I think it might be worth to add a comment here, on why are we doing this.
e.g. `// Only enforce event limit on newly created event definitions`

Reply:
added some comments